Misc/NEWS.d/next/Library/2024-10-15-16-50-03.gh-issue-125541.FfhmWo.rst

```diff
@@ -0,0 +1,3 @@
+Pressing :kbd:`Ctrl-C` while blocked in :meth:`threading.Lock.acquire` now
```

Comment:
Does this also apply to RLock?

Reply:
Yes. I think it also applies to `Thread.join()`